amethyst_ui/src/font/default.rs

```diff
@@ -18,11 +18,16 @@ pub fn get_default_font(loader: &Loader, storage: &AssetStorage<FontAsset>) -> F
         Ok(handle) => match handle {
             FontKitHandle::Path { path, .. } => {
                 if let Some(file_extension) = path.extension() {
-                    let format = if file_extension == "ttf" || file_extension == "otf" {
-                        Some(TtfFormat)
-                    } else {
-                        error!("System font '{}' has unknown format", path.display());
-                        None
+                    let format = match file_extension.to_str() {
+                        Some(ext) => {
+                            let ext = ext.to_lowercase();
```

Comment:
Note: this could use [`eq_ignore_ascii_case`](https://doc.rust-lang.org/std/primitive.str.html#method.eq_ignore_ascii_case) to avoid the copies used in `to_lowercase`.

Reply:
Interesting! I'll try that.